TicketingSystem.Common/Model/Database/Enums/SeatStatus.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace TicketingSystem.Common.Model.Database.Enums
+{
+    public enum SeatStatus
+    {
+        Free = 1,
```

Comment:
Default value for enum is `0`, so it's not a good idea to start with `1`

Reply:
Changed it to 0